configs/minimal_eng.yaml

```diff
@@ -34,9 +33,35 @@ n_train: 5
 n_val: 5
 batch_size: 1
 max_epochs: 10
+learning_rate: 0.0
```

Comment:
?

Reply:
??!! I don't know how that happened... 🤦‍♀️ I'll check out the original version